doc/api/readline.md

```diff
@@ -318,24 +318,22 @@ changes:
     description: Symbol.asyncIterator support is no longer experimental.
 -->
 
-> Stability: 2 - Stable
-
 * Returns: {AsyncIterator}
 
 Create an `AsyncIterator` object that iterates through each line in the input
 stream as a string. This method allows asynchronous iteration of
-`readline.Interface` objects through `for`-`await`-`of` loops.
+`readline.Interface` objects through `for await...of` loops.
 
 Errors in the input stream are not forwarded.
 
 If the loop is terminated with `break`, `throw`, or `return`,
 [`rl.close()`][] will be called. In other words, iterating over a
 `readline.Interface` will always consume the input stream fully.
 
-A caveat with using this experimental API is that the performance is
-currently not on par with the traditional `'line'` event API, and thus it is
-not recommended for performance-sensitive applications. We expect this
-situation to improve in the future.
+A caveat with using this API is that the performance is currently not on par
+with the traditional `'line'` event API, and thus it is not recommended for
+performance-sensitive applications. This situation is expected to improve in the
+future.
```

Comment:
I'd remove the last sentence. "This will be better at some undisclosed point in the future, maybe" is better left unsaid IMO. When the situation improves, docs can be updated. (In general, we should avoid relative-time statements like "currently" and "in the future".)